tst/EnergyPlus/api/TestRuntime.c

```diff
@@ -134,48 +134,53 @@ void newEnvrnHandler(EnergyPlusState state)
     printf("NEW ENVIRONMENT CALLBACK: Starting a new environment\n");
 }
 
-void progressHandler(EnergyPlusState state, int const progress)
+void progressHandler(int const progress)
```

Comment:
Cleaned up this runtime test script, fixing some things along the way and adding test coverage for the runtime.cc file.